### ISSUE TITLE
About depth map of SGD dataset 

### ISSUE THREAD

**Issue by ader47** [wenquanlu/HandRefiner#37]
Hi! @wenquanlu 
I‘ve noticed that the  accuracy of the depth map in Static Gesture Dataset (SGD) is meter level. During training, should we use the ground truth of the depth map or the depth map converted from the normal map? Thanks! 🙏

**Comment by wenquanlu** [comment]
Hi! we used the ground truth of depth map provided in the SGD.

There is one preprocessing step though, We apply normalization to the depth map of each hand in both training sets and the HandRefiner pipeline. On a scale of 0 (black) to 1 (white), the closest surface is given the value of 1.0, and the furthest surface is given the value of 0.2.

**Comment by ader47** [comment]
> Hi! we used the ground truth of depth map provided in the SGD.
> 
> There is one preprocessing step though, We first segment out the hand depth map and apply normalization to the depth map. On a scale of 0 (black) to 1 (white), the closest surface is given the value of 1.0, and the furthest surface is given the value of 0.2.
> 
> Hope this helps.

OK, thank you so much!

**Comment by wenquanlu** [comment]
<img width="238" alt="training_procedure" src="https://github.com/wenquanlu/HandRefiner/assets/64842615/39b474df-fa70-44c6-9f88-e3ebe9b3590a">

This is the procedure to process training set for RHD. SGD is very similar to this
